src/www/app/outline_server_repository/server.ts

```diff
@@ -40,7 +40,7 @@ export class OutlineServer implements Server {
     private net: NativeNetworking,
     private eventQueue: events.EventQueue
   ) {
-    this.config = accessKeyToShadowsocksConfig(accessKey);
+    this.config = accessKeyToServerConfig(accessKey);
```

Comment:
In the future, this will go from access key to the static or dynamic config. But for now, let's stick to the `ShadowsocksSessionConfig`

Reply:
Yep, that's the next PR I'm planning.